ui/src/kapacitor/components/RuleHandlers.tsx

```diff
@@ -171,6 +171,11 @@ class RuleHandlers extends PureComponent<Props, State> {
       ...selectedItem,
       alias: newItemName,
     }
+
+    if (newEndpoint.type === AlertTypes.kafka) {
```

Comment:
could you add a comment giving some context for this? it looks like a hack for inconsistency in kapacitor

Reply:
That it is.